src/custom-surf/pages/ServiceTicketOpen.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019-2022 SURF.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { EuiPage, EuiPageBody, EuiText } from "@elastic/eui";
+import Explain from "components/Explain";
+import OpenForm from "custom/components/cim/OpenForm";
+import { OpenServiceTicketPayload, ServiceTicket } from "custom/types";
+// import { CreateServiceTicketPayload } from "custom/types";
+import React, { useContext, useState } from "react";
+import { useParams } from "react-router";
+import ApplicationContext from "utils/ApplicationContext";
+
+interface IProps {
+    id: string;
+}
+
+export default function ServiceTicketOpen() {
+    const { id } = useParams<IProps>();
+    const [showExplanation, setShowExplanation] = useState(false);
+    const { redirect, customApiClient } = useContext(ApplicationContext);
+
+    const handleSubmit = (userInputs: any) => {
+        const payload: OpenServiceTicketPayload = {
+            cim_ticket_id: id,
+            title_nl: userInputs.title_nl,
+            description_nl: userInputs.description_nl,
+            title_en: userInputs.title_en,
+            description_en: userInputs.description_en,
+            mail_subject: userInputs.mail_subject,
+        };
+        // TODO:
+        //   - ask for confirmation here?
+        //   - or suggestion by Rene, redirect to other page for displaying the emails?
```

Comment:
todo; check if we can have a simple popup here that describes what's going to happen

Reply:
Submitting the underlying form from a popup modal proved too difficult for me. Workaround is that the final form submit button has a different color and explicit text `Going to send mail to <number> institutes`.